modules/microshift-kickstart-rpm-install.adoc

```diff
@@ -43,9 +43,9 @@ $ export RHSM_KEY="$(cat ~/.rhsm-activation-key)"
 +
 [source,terminal]
 ----
-$ export MICROSHIFT_VER=4.17 # <1>
+$ export MICROSHIFT_VER={product-version} # <1>
 ----
-<1> For the minor version you set, the latest z-stream of that version is installed.
+<1> The latest z-stream of the minor version that you set is installed.
```

Comment:
Consider:
```suggestion
<1> The latest `.z` version of the minor version that you set is installed.
```
(matches text above it, seems a little less jargony than "z-stream" to me)